backend/src/openarchiefbeheer/conf/base.py

```diff
@@ -86,6 +86,10 @@
     },
 }
 
+# Geospatial libraries
+GEOS_LIBRARY_PATH = config("GEOS_LIBRARY_PATH", None)
+GDAL_LIBRARY_PATH = config("GDAL_LIBRARY_PATH", None)
```

Comment:
Why are these needed?

Reply:
For generating the docs, it's a Mac thing where, when installed with brew (package manager) certain system libraries may have non-standard paths. This allows to configure them using a .env, similar to: https://github.com/open-zaak/open-zaak/commit/6cbb045195f817d9bf00cca3c66938c55a98e8c2